timescaledb/how-to-guides/continuous-aggregates/compression-on-continuous-aggregates.md

```diff
@@ -0,0 +1,68 @@
+# Compression on continuous aggregates
+Continuous aggregates are often used to store down sampled historical data.
+The historical data is almost never modified or recomputed and is only used 
+for serving analytic queries. For this use case, it is often beneficial to 
+store the materialized data in compressed form to save on storage costs. 
+You can get these cost savings by enabling compression on continuous 
+aggregates.
+
+At this time, TimescaleDB does not support refreshing compressed regions of a 
+continuous aggregate. In order to do this, you will have to manually decompress 
+the compressed chunk and then execute a `refresh_continuous_aggregate` call.
+
+## Enable compression on continuous aggregates
+You can enable and disable compression on continuous aggregated by setting
+`compress` parameter when you alter the view.
+
+<procedure>
+
+### Enabling/Disabling compression on continuous aggregate aggregation
+1.  For an existing continuous aggregate, at the `psql` prompt, enable
+ compression:
+
+    ```sql
+    ALTER MATERIALIZED VIEW cagg_name set (timescaledb.compress = true);
+    ```
+1.  Disable compression. 
+
+    ```sql
+    ALTER MATERIALIZED VIEW cagg_name set (timescaledb.compress = false);
+    ```
+    
+    This command fails if there are compressed chunks associated with the 
+    continuous aggregate. If there are compressed chunks, first
+    decompress the chunks and then drop any compression policy on the continuous
+    aggregate, before trying to disable compression. For
+    more detailed information, see [Decompress chunks] [decompress-chunks]
+  
+    ```sql
+    SELECT decompress_chunk(c, true) FROM show_chunks('cagg_name') c;
+    ```
+ 
+</procedure>
+
+## Compression policies on continuous aggregates
+Before  setting up a compression policy on a continuous aggregate, you should
+set up a refresh policy. The compression policy interval should be set so that
+actively refreshed regions are not compressed. This is to prevent refresh
+policies from failing.  To illustrate, with a refresh policy like so:
+
+```sql
+SELECT add_continuous_aggregate_policy('cagg_name', refresh_start=>'30 days', refresh_end=>'1 day', '1 h');
+```
+
+The compression policy should be set so that `compress_after` parameter is greater than the `refresh_start` parameter of the continuous aggregate policy.
+
```

Comment:
```suggestion
With this kind of refresh policy, the compression policy needs the `compress_after` 
parameter greater than the `refresh_start` parameter of the continuous aggregate policy:
```

Reply:
Its not specific to this example. 
compress_after should always be greater than refresh_start. 
The example is intended to illustrate how to go about doing that.

Reply:
```suggestion
We recommend that the `compress_after` parameter is greater than the `refresh_start` parameter of the continuous aggregate policy:

```

Reply:
Well, it throws an error if it does not meet the condition. That's the reason I phrased it as should.

Reply:
```
test=# select add_compression_policy('metrics_cagg', ' 1 h'::interval);
ERROR:  compress_after value for compression policy should be greater than the start of the refresh window of continuous aggregate policy for metrics_cagg
```

Reply:
```suggestion
The `compress_after` parameter must be greater than the `refresh_start` parameter of the continuous aggregate policy:

```